src/frequenz/sdk/actor/_config_managing.py

```diff
@@ -78,8 +79,8 @@ async def run(self) -> None:
         """
         await self.send_config()
 
-        async for path in self._file_watcher:
-            if str(path) == self._conf_file:
+        async for event in self._file_watcher:
+            if str(event.path) == self._conf_file:
```

Comment:
Now that we also have the [event type](https://frequenz-floss.github.io/frequenz-channels-python/v0.16/reference/frequenz/channels/util/#frequenz.channels.util._file_watcher.FileWatcher.Event.type) available, we should check what the type is, and should not try to read the file if it is a [`DELETE` event](https://github.com/frequenz-floss/frequenz-channels-python/blob/a4b873ba8c3b38df78ce8f6883dc82d143894859/src/frequenz/channels/util/_file_watcher.py#L29), as it will fail.

This is a pre-existing issue, so we can address it in a separate PR too, please create a new issue if you prefer to do that. If you prefer to fix it in this PR, please do the fix in a separate commit.